.github/workflows/ci.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Rust Toolchain
+        run: rustup default stable
```

Comment:
use the rust toolchain action

Reply:
how?

Reply:
@madninja The previous troubles were with the Rust action. This is the only way that has worked so far.